sympy/core/sympify.py

```diff
@@ -273,6 +272,31 @@ def sympify(a, locals=None, convert_xor=True, strict=False, rational=False,
     >>> kernS(s)
     -2*(-(-x + 1/x)/(x*(x - 1/x)**2) - 1/(x*(x - 1/x))) - 1
 
+    Parameters
+    ==========
+
+    a : Can be :
+        - any object defined in SymPy
+        - standard numeric python types: int, long, float, Decimal
+        - strings (like "0.09" or "2e-19")
+        - booleans, including ``None`` (will leave ``None`` unchanged)
+        - dict, lists, sets or tuples containing any of the above
+
+    convert_xor : boolean, optional
+        If true, treats XOR, $^$, as exponentiation, $**$. Used when input is a string
+
+    locals : any object defined in SymPy
+        In order to have strings be recognized it can be imported into a namespace dictionary and passed as locals
+
+    strict : boolean, optional
+        If the option strict is set to True, only the types for which an explicit conversion has been defined are converted. In the other cases, a SympifyError is raised
+
+    rational : boolean, optional
+        If true, converts floats into Rational. Used when input is a string.
+
+    evaluate : boolean
+        If False, then arithmetic and operators will be converted into their SymPy equivalents
```

Comment:
Can you also add what happens if `rational = True` and `evaluate = False`?

Reply:
Done. I also think demonstrating the differences in the form of examples would be better. I can add examples too.

Reply:
Sure, you can add some examples. great idea.

Reply:
Done!